### PR TITLE
Fix Serverfarm deployment ARM template to meet newer requirements

### PR DESCRIPTION
**Pull request by CodeYouMust**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Fix Serverfarm deployment ARM template to meet to newer requirements.
* Fix numbering of steps

### CODE REVIEW

**Comment by CodeYouMust** [comment]
Removing GovCloud because the CI/CD for this pipeline `AzQUickStarts-USGov` fails with error that resources are not available in GovCloud. Tried various SKUs for app service. No UsGov consumers use this integration. Will re-add it once CICD pipeline has capacity to test it.

```
5:45:50 PM - The deployment 'mainTemplate-deployment' failed with error(s). Showing 2 out of 2 error(s).

Status Message: No available instances to satisfy this request. App Service is attempting to increase capacity. Please retry your request later. If urgent, this can be mitigated by deploying this to a new resource group. (Code: Conflict)

 - No available instances to satisfy this request. App Service is attempting to increase capacity. Please retry your request later. If urgent, this can be mitigated by deploying this to a new resource group. (Code:)
 -  (Code:Conflict)
 -  (Code:)
```
[Link to error in CICD](https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=110093&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=97b1930a-cbd5-5206-3c42-940d27a1bdc3&l=442)

File: quickstarts/microsoft.apimanagement/api-management-logs-to-moesif-using-eventhub-webapp/metadata.json [hunk 1]
```diff
@@ -4,10 +4,9 @@
   "description": "The template will log API calls from Azure API Management to Moesif API analytics so you can understand customer API usage and resolve customer issues quickly.",
   "summary": "Powerful API analytics for Azure API Management to help you activate, understand, and monetize customers",
   "githubUsername": "Moesif",
-  "dateUpdated": "2022-06-08",
+  "dateUpdated": "2022-08-30",
   "type": "QuickStart",
   "environments": [
-    "AzureUSGovernment",
```
Comment (bmoore-msft) [inline quickstarts/microsoft.apimanagement/api-management-logs-to-moesif-using-eventhub-webapp/metadata.json:10]: Why are we removing USGov support?  (and not the other sovereign clouds?)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.apimanagement/api-management-logs-to-moesif-using-eventhub-webapp/metadata.json (hunk 1 above)
Comment (bmoore-msft) [reply quickstarts/microsoft.apimanagement/api-management-logs-to-moesif-using-eventhub-webapp/metadata.json:10]: Ok, I see the post above - that a transient problem though, not a statement of support.  If we deploy to usgovtexas does that work?

Comment (CodeYouMust) [reply quickstarts/microsoft.apimanagement/api-management-logs-to-moesif-using-eventhub-webapp/metadata.json:10]: Hi Thx for feedback. Somehow I missed your later comment, and based on first comment, I just pushed change to remove `AzureGermanCloud` and `AzureChinaCloud`. This quickstart is probably not getting used by any of sovereign clouds.

Yes, it does look like a `transient` problem, however, it has failed every run today (4 runs i think), and without this change, mutual customers are blocked from utilizing functionality. I am not sure how to use `usgovtexas`.

Request we merge this change, and then after the govcloud issues are resolved, we can easily re-add back the sovereign clouds.

**Comment by DanielleDigman56** [comment]
Hello,I have not replied or made any comments.So I was suspicious that my phone was hacked.Please do not make any changes or run any programs.I am not wavy with programming.so I really have no clue as to what is default settings and what is not.I believe the person who has access to my phone and accounts is listing themself as a parent.Do you have any suggestions of how to remove programs that have access to my data and activity?I really should reset my phone but I’ve been dealing with the issue of things not getting backed or disappearing from data downloads.If you have any suggestions of programs to get rid of things that have been installed I would greatly appreciate it.Thank you

**Comment by CodeYouMust** [comment]
@bmoore-msft 
Looks like `Azure Govcloud` transient issue from last week is no longer there.. so I updated PR . Looks like everything passes, no more readme-violations.

**Pull request merged**